### ISSUE TITLE
the red color in ViewRenderable display as orange

### ISSUE THREAD

**Issue by lgz800** [SceneView/sceneform-android#252]
hi
 I use sceneform 1.20.2 version, it depend on filament 1.12.8 version
 I made a ViewRenderable inlucde a red circle image. but in screen it display as orange.

<img src = 'https://s3.bmp.ovh/imgs/2021/12/02b089bb430891e8.jpg' />

when i change filament into 1.12.1 version,  the color will be well.  

<img src = 'https://s3.bmp.ovh/imgs/2021/12/2dc4ed82e4435607.jpg' />

And filament 1.12.2 version is same with filament 1.12.8.
Is there any adaptation problem? Thank you :)

imageview_card,xml is:
```
<?xml version="1.0" encoding="utf-8"?>
<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:tag="image"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    android:src="@drawable/ad_act_3"
    android:background="#ff0000"
    tools:ignore="MissingDefaultResource">
</ImageView>
```

ad_act_3 is:
<img src = 'https://s3.bmp.ovh/imgs/2021/12/1c35e66c2cf72ca6.png' />

create viewrenderable codes is:
```
ViewRenderable.builder()
                .setView(this, R.layout.imageview_card)
                .build()
                .thenAccept(
                        (renderable) -> {
                            renderable.setSizer(new FixedHeightViewSizer(0.5f));
                            ViewRenderable imageRenderable = renderable;
                            imageRenderable.setVerticalAlignment(ViewRenderable.VerticalAlignment.CENTER);
                            imageRenderable.setHorizontalAlignment(ViewRenderable.HorizontalAlignment.CENTER);
                            MainActivity activity = weakActivity.get();
                            if (activity != null) {
                                activity.model = renderable;
                            }
                        })
                .exceptionally(
                        (throwable) -> {
                            throw new AssertionError("Could not load plane card view.", throwable);
                        });
```

**Comment by RGregat** [comment]
Maybe something has changed again on the tone mapping. The current version of Filament is 1.15.1 which is on our to do list for an upcomming update.

**Comment by grassydragon** [comment]
This solution can be used for now to get the correct colors: https://github.com/SceneView/sceneform-android/issues/295#issuecomment-1024864788

**Comment by RGregat** [comment]
@lgz800 do you have still the problem? Can you test your usecase with the latest version of SceneForm?

**Comment by ThomasGorisse** [comment]
Fixed: #311
